### ISSUE TITLE
Hidden pcr-apps cause scrollbar when resizing window

### ISSUE THREAD

**Issue by EzraZebra** [Simonwep/pickr#120]
#### What is the current behavior?
When resizing the window, the pcr-app doesn't reposition until it's shown. This can cause scrollbars to appear.

#### Reproduction
Can be reproduced on the template fiddle: [JSFiddle](https://jsfiddle.net/Simonwep/qx2Lod6r/).
Simply resize the running fiddle until a horizontal scrollbar appears, but keep it wide enough to fit the pcr-app.  
Then click the button, and the scrollbar will disappear.

#### What is the expected behavior?
Hidden pcr-apps should reposition as the window is resized, to avoid scrollbars appearing and disappearing.

#### Environment:
```
Version/bundle/theme: the ones used in the JSFiddle template
Browser-version:  Firefox 68 x64
Operating-system:  Windows 8.1
```

**Comment by Simonwep** [comment]
Oh yeah, you're right. Fixed in 77b32cd01503f803e612aa4126cdb18e1bda8561 :)

**Comment by EzraZebra** [comment]
Cool, thanks!
